sys/net/network_layer/ipv4/addr/ipv4_addr.c

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2022 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for
+ * more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ *
+ * @author      Hendrik van Essen <hendrik.ve@fu-berlin.de>
+ */
+
+#include <assert.h>
+#include <stdlib.h>
+#include <string.h>
+#include <stdio.h>
+
+#include "fmt.h"
+#include "kernel_defines.h"
+#include "net/ipv4/addr.h"
+
+void ipv4_addr_print(const ipv4_addr_t *addr)
+{
+    assert(addr);
+    char addr_str[IPV4_ADDR_MAX_STR_LEN];
+    ipv4_addr_to_str(addr_str, addr, sizeof(addr_str));
```

Comment:
Where is this function implemented? Wouldn't it make sense to add `ipv4_addr_print()` there as well?

Reply:
`ipv4_addr_to_str` can be found in `net/ipv4/addr.h` like `ipv6_addr_to_str` can be found in `net/ipv6/addr.h`. I'm just placing the IPv4 variants relative to their respective IPv6 counterparts ;)

Reply:
In `net/ipv6/addr.h` there is just the definition of `ipv4_addr_to_str()`, but I now saw, that they are in their own compile unit `ipv6_addr_to_str.c`. So nvm.